api/proto/teleport/authinfo/v1/authinfo.proto

```diff
@@ -0,0 +1,37 @@
+// Copyright 2025 Gravitational, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+package teleport.authinfo.v1;
+
+import "teleport/header/v1/metadata.proto";
+
+option go_package = "github.com/gravitational/teleport/api/gen/proto/go/teleport/authinfo/v1;authinfo";
+
+// AuthInfo holds meta-information for specific server.
+message AuthInfo {
+  string kind = 1;
+  string sub_kind = 2;
+  string version = 3;
+  teleport.header.v1.Metadata metadata = 4;
+
+  AuthInfoSpec spec = 5;
+}
+
+// AuthInfoSpec encodes the parameters auth server meta-information.
+message AuthInfoSpec {
+  // teleport_version advertises the version of the auth server, e.g. "17.3.3".
```

Comment:
I would consider stressing that this does not have a leading "v", i.e. it's a string that's valid according to coreos/go-semver.